src/components/slider/slider.e2e.ts

```diff
@@ -673,4 +673,49 @@ describe("calcite-slider", () => {
   it("is form-associated", () => formAssociated("calcite-slider", { testValue: 5 }));
 
   it("is form-associated with range", () => formAssociated("calcite-slider", { testValue: [5, 10] }));
+
+  it("groupSeparator: displays numbers as comma separated when prop is true and does not when false", async () => {
+    const page = await newE2EPage({
```

Comment:
For consistency, can you create an empty page and use `setContent` afterwards? No need for `waitForChanges` after doing this.

Reply:
Oops, sorry.